client/src/app.js

```diff
@@ -239,7 +253,10 @@ class App extends Component {
             marginTop: 0
           }}
         >
-        <div className='overlay-buttons'>
+
+        <div className='overlay-buttons'style={{
+                display: saveable ? 'block' : 'none'
```

Comment:
This function can still be triggered even if the button is not visible, which is a problem.